src/simplicial-hott/05-segal-types.rzk.md

```diff
@@ -216,6 +216,66 @@ Extension types are also used to define the type of commutative triangles:
         t₂ ≡ t₁ ↦ h t₂]   -- the diagonal is exactly `h`
 ```
 
+## Arrow types
+
+We define the arrow type:
+
+```rzk
+#def arr
+  ( A : U)
+  : U
+  := Δ¹ → A
+```
+
+For later convenience we give an alternative characterizations of the arrow
+type.
+
+```rzk
+#def fibered-arr
+  ( A : U)
+  : U
+  := Σ (x : A) , (Σ (y : A) , hom A x y)
+
+#def fibered-arr-free-arr
+  ( A : U)
+  : (arr A) → (fibered-arr A)
+  := \ k → (k 0₂ , (k 1₂ , k))
+
+#def is-equiv-fibered-arr-free-arr
+  ( A : U)
+  : is-equiv (arr A) (fibered-arr A) (fibered-arr-free-arr A)
+  :=
+    ( ( \ (_ , (_ , f)) → f , \ _ → refl)
+    , ( \ (_ , (_ , f)) → f , \ _ → refl))
```

Comment:
```suggestion
    ( ( (\ (_ , (_ , f)) → f) , (\ _ → refl))
    , ( (\ (_ , (_ , f)) → f) , (\ _ → refl)))
```

@fredrik-bakke do I undestand correctly, that you prefer adding the brackets like this (and in the other place)?

Reply:
Ah, hmm, I guess here you _could_ select to omit the highlighted parentheses if you want to, as it is consistent on both sides of the comma and the total pairing expression is on a single line.